app/Modules/Availability/AvailabilityService.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace App\Modules\Availability;
+
+use App\Constants\ControllerRating;
+use App\Exceptions\OverlappingAvailabilityException;
+use App\Exceptions\PrivilegeException;
+use App\Modules\Availability\AvailabilityInPastException;
+use App\Modules\Position\Position;
+use App\User;
+use Carbon\Carbon;
+
+class AvailabilityService
+{
+    /** @var mixed App\User */
+    protected $user;
+
+    /** @var int THIS MINIMUM TIME AVAILABILITY SHOULD BE */
+    public $TIME_IN_MINS = 30;
+
+    public function __construct(User $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Check if the the user is able to book the given position only on the rating requirement.
+     *
+     * @param  User  $user
+     * @param  Position  $position
+     * @return bool
+     */
+    public function validateRatingRequirement(User $user, Position $position): bool
```

Comment:
This method shouldn't be necessary for adding availability.

Reply:
done